core/api/core.api

```diff
@@ -206,13 +206,21 @@ public final class com/revenuecat/purchases/kmp/Purchases$Companion {
 	public final fun setSimulatesAskToBuyInSandbox (Z)V
 }
 
+public final class com/revenuecat/purchases/kmp/PurchasesAreCompletedBy : java/lang/Enum {
+	public static final field MY_APP Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;
+	public static final field REVENUECAT Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;
+	public static fun getEntries ()Lkotlin/enums/EnumEntries;
+	public static fun valueOf (Ljava/lang/String;)Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;
+	public static fun values ()[Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;
+}
+
 public final class com/revenuecat/purchases/kmp/PurchasesConfiguration {
-	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/String;ZLjava/lang/String;ZLcom/revenuecat/purchases/Store;ZLcom/revenuecat/purchases/kmp/DangerousSettings;Lcom/revenuecat/purchases/kmp/EntitlementVerificationMode;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/String;Lcom/revenuecat/purchases/kmp/PurchasesAreCompletedBy;Ljava/lang/String;ZLcom/revenuecat/purchases/Store;ZLcom/revenuecat/purchases/kmp/DangerousSettings;Lcom/revenuecat/purchases/kmp/EntitlementVerificationMode;Lkotlin/jvm/internal/DefaultConstructorMarker;)V
 	public final fun getApiKey ()Ljava/lang/String;
 	public final fun getAppUserId ()Ljava/lang/String;
 	public final fun getDangerousSettings ()Lcom/revenuecat/purchases/kmp/DangerousSettings;
 	public final fun getDiagnosticsEnabled ()Z
-	public final fun getObserverMode ()Z
```

Comment:
I do wonder if we would want to deprecate these instead of removing.... But not sure

Reply:
I'm not sure, but wouldn't it be unusual to release a library with already-deprecated things? It would make it "on par" with the platform SDKs, but I don't know how many developers will be copy-pasting their Android implementation code to KMP, for instance. (In that scenario it would be beneficial to have as much parity with the platform SDKs as possible.)

Reply:
Well, at the end of the day I think is to try to not break developers that might already be using this... I think it's fair game to do breaking changes in a 0.X version, but the easier we make it for devs the better. So yeah, I think it's not worth it, please ignore me :P

Reply:
Good point! Unfortunately this is not the only breaking change. I'm going to write a migration guide, and will shout out on Twitter that I'll help anyone migrating 😄